docs/csharp/language-reference/keywords/out.md

```diff
@@ -18,6 +18,36 @@ You can use the `out` keyword in two contexts:
 
 - In [generic type parameter declarations](out-generic-modifier.md) for interfaces and delegates, which specifies that a type parameter is covariant.
 
+## Example
+The out keyword can be especially useful when you want to get 2 values back from a method for 2 local variables, without using a custom object, tuple or class-scoped variable. In the below example, the method `IsDivisibleBy2` takes in an integer, determines if it is divisible by 2, returns the result as a `bool` and outs the division result.
```

Comment:
I'd soften this somewhat:

```suggestion
You can use the `out` keyword to retrieve multiple values back from a method in multiple variables, without using a custom object, tuple or class-scoped variable. In the below example, the method `IsDivisibleBy2` takes in an integer, determines if it is divisible by 2, returns the result as a `bool` and outs the division result.
```

Reply:
I agree with you on other points, but disagree with you on this wording. In my opinion the former is better.